samples/wifi/scan/README.rst

```diff
@@ -20,9 +20,21 @@ Overview
 ********
 
 This sample can perform Wi-Fi scan operations in the 2.4GHz and 5GHz bands.
+The default scan type is active scan. However, you can configure the scan type to be either active or passive.
 
 Using this sample, the development kit can scan for available access points in :abbr:`STA (Station)` mode.
 
+|config|
+
+You may configure the Wi-Fi scan type in the :file:`prj.conf` file:
+
+* Scan type
```

Comment:
Do we need this line?

Reply:
No removed this line, as refernce in prj.conf is no more. Updated doc accordingly.